drivers/led_strip/Kconfig.ws2812

```diff
@@ -46,4 +46,11 @@ config WS2812_STRIP_GPIO
 	  Note that this driver is not compatible with the Everlight B1414
 	  controller.
 
+config WS2812_STRIP_RPI_PICO_PIO
+	bool "Raspberry Pi Pico PIO"
+	depends on DT_HAS_WORLDSEMI_WS2812_RPI_PICO_PIO_ENABLED
```

Comment:
You can add

<pre>
default y
</pre>

since it depends on the dts node status value.

Reply:
Changed it to DBG.

Reply:
As a side note the `led_strip` drivers have not been converted to the DT node availability + `default y` mechanism.

In addition we had some complains about the automagic LED driver selection: https://github.com/zephyrproject-rtos/zephyr/issues/54609.